src/components/progress-bar/ProgressBar.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import './ProgressBar.scss';
+
+type Props = {
```

Comment:
```suggestion
export interface ProgressBarProps {
```

Reply:
how interface is better than type and we should we export it now?
please note there `type` is used in current implementations

Reply:
Thats a good question, it specified in the documentation for TS migration but I do not know why because I joined after the decision.

Reply:
We won't avoid type aliases because that's the only to do unions, for example so it's not like we're going to be consistent anyway. I would enforce one over other tbh. 
I've change to interface as you wished.